src/main/java/com/taboola/async_profiler/api/facade/ProfileRequest.java

```diff
@@ -1,18 +1,17 @@
 package com.taboola.async_profiler.api.facade;
 
-import lombok.Data;
-
-import java.util.Objects;
-
 import com.taboola.async_profiler.api.original.Events;
 
+import lombok.Data;
+
 @Data
 public class ProfileRequest {
 
     private int durationSeconds = 60;//profiling duration
     private int frameBufferSize = 5_000_000;
     private Integer samplingIntervalMs = 10;//10ms default, relevant only for non alloc events.
     private Integer samplingIntervalBytes = 10_000_000;//relevant only for alloc event.
+    private boolean separateThreads = false;
```

Comment:
if you're already here, can you please change samplingIntervalMs's default to 1?